Fix #5370, part of #59: Migrate to Bazel 6.5.0 (#4886)
oppia/oppia-android: wiki/Bazel-Setup-Instructions-for-Windows.md

At this point, your system should be able to build Oppia Android. To verify, try building the APK (from your subsystem terminal -- note that this & all other Bazel commands must be run from the root of the ‘oppia-android’ directory otherwise they will fail):

```sh

> bazel build //:oppia_dev

Author: ```suggestion
bazel build //:oppia
```

Author: Addressed.

```

(Note that this command may take 10-20 minutes to complete depending on the performance of your machine).

If everything is working, you should see output like the following:

```

> Target //:oppia_dev up-to-date:
>   bazel-bin/oppia_dev.aab

Author: ```suggestion
Target //:oppia up-to-date:
  bazel-bin/oppia_deploy.jar
  bazel-bin/oppia_unsigned/apk
  bazel-bin/oppia/apk
```

Author: Addressed.

INFO: Elapsed time: ...
INFO: 1 process...
INFO: Build completed successfully, ...
```

If you see the above, you can proceed to use your subsystem for Bazel commands while developing Oppia. If you instead see an error, please [file an issue](https://github.com/oppia/oppia-android/issues/new/choose).

> Note also that the ``oppia_dev.aab`` under the ``bazel-bin`` directory of your local copy of Oppia Android should be a fully functioning development version of the app that can be installed using bundle-tool. However, it's recommended to deploy Oppia to an emulator or connected device using the following Bazel command:
> 
> ```sh
> bazel run //:install_oppia_dev
> ```
> 
> If this fails, you may need to use ADB from within a Windows command prompt and install ``bazel-bin/oppia.apk`` which can be produced via the following command to build an APK directly:
> 
> ```sh
> bazel build //:oppia
> ```

Author: ```suggestion
Note also that the ``oppia.apk`` under the ``bazel-bin`` directory of your local copy of Oppia Android should be a fully functioning development version of the app that can be installed using ``adb`` (though we recommend using ADB from within a Windows command prompt or shell since the Ubuntu subsystem may not have correct support to access devices or emulators connected to the native Windows machine).
```

Author: Addressed.
